annotationProcessor/src/main/java/org/corfudb/annotations/ObjectAnnotationProcessor.java

```diff
@@ -0,0 +1,642 @@
+package org.corfudb.annotations;
+
+import com.google.common.collect.ImmutableMap;
+import com.squareup.javapoet.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.corfudb.runtime.object.*;
+
+import javax.annotation.processing.*;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.*;
+import javax.lang.model.type.DeclaredType;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+import javax.tools.Diagnostic;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.*;
```

Comment:
At some point we need to following a coding style standard, it will only get worse going forward.
Maybe re-enable our checkstyle ?

How to disable import wild card imports
fyi: http://stackoverflow.com/a/25916603